### PR TITLE
Feature/New vehicle data `handsOffSteering`

### PR DESCRIPTION
**Pull request by Mariia-Novosiadla**
Guidelines updates according to issue [#205](https://github.com/smartdevicelink/rpc_spec/issues/205).
 [#3126](https://github.com/smartdevicelink/sdl_core/issues/3126)

Proposal [SDL-0257](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0257-New-vehicle-data-HandsOffSteering.md)

This PR is **ready** for review.

### Summary

1.  Updates for **Common**
* extending `VehicleDataType` with new element `VEHICLEDATA_HANDSOFFSTEERING`

2. Updates in **VehicleInfo**
* adding `handsOffSteering` to request/response params list of the following RPCs:
SubscribeVehicleData
UnsubscribeVehicleData
GetVehicleData
OnVehicleData

### CLA
* [x]  I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by LitvinenkoIra** [comment]
@atiwari9 please review this PR

**Comment by atiwari9** [comment]
PR Approved

**Comment by LitvinenkoIra** [comment]
@theresalech PR is ready for Livio review. Thank you!

File: docs/VehicleInfo/SubscribeVehicleData/index.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Purpose
 |engineOilLife|Boolean|false||
 |electronicParkBrakeStatus|Boolean|false||
 |cloudAppVehicleID|Boolean|false||
+|handsOffSteering|Bolean|false||
```
Comment (jacobkeeler) [inline docs/VehicleInfo/SubscribeVehicleData/index.md:47]:
```suggestion
|handsOffSteering|Boolean|false||
```

Comment (Mariia-Novosiadla) [reply docs/VehicleInfo/SubscribeVehicleData/index.md]: fixed in aa1b73342f4e28b176864bd248d88635519290bc

**Comment by AKalinich-Luxoft** [comment]
@jacobkeeler conflicts have been resolved here

**Pull request merged**